docs/zephyr_02_functional_requirements.sdoc

```diff
@@ -1723,19 +1762,45 @@ Zephyr shall provide an interface to schedule multiple threads.
 DISCUSSION_DATE: >>>
 pf-ok
 <<<
+RELATIONS:
+- TYPE: Parent
+  VALUE: ZEP-5
 
 [REQUIREMENT]
-UID: ZEP-121
+UID: ZEP-125
 STATUS: Draft
 TYPE: Functional
 COMPONENT: Threads
-TITLE: Threads - TBD
+TITLE: Thread Options
 STATEMENT: >>>
-??? Can Zephyr change priviledge level of a thread once created??
+The Zephyr kernel shall support a set of thread options.
```

Comment:
**question:** What is that hinting at? What application level problem does that solve?

Reply:
It solves the option that threads can have special treatment like a user thread or an essential thread.